src/kili/mutations/user/__init__.py

```diff
@@ -90,23 +89,14 @@ def update_password(
         result = self.auth.client.execute(GQL_UPDATE_PASSWORD, variables)
         return format_result("data", result)
 
-    @typechecked
+    @deprecate(msg="This method is deprecated. Please use the Kili App to reset your password.")
```

Comment:
Deprecated means that we will remove it in the future. So we should rather throw an Exception saying that this method has been removed.